react/useCountry.ts

```diff
@@ -0,0 +1,13 @@
+import { useRuntime } from 'vtex.render-runtime'
+import { useAddressContext } from 'vtex.address-context/AddressContext'
+
+export const useCountry = () => {
+  const { culture } = useRuntime()
+  const { address } = useAddressContext()
+
+  // TODO: should also try to get country from:
+  // - addresses in account
+  // - IP
+  // - store config
```

Comment:
This comments already existed and I just bring them here. Does anyone knows if we have jira tasks for those already?

Reply:
we do not, also I think "store config" is the one we get from `culture.country` already